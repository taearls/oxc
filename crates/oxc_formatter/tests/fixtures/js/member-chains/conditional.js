id === null
  ? null
  : internal.getSuspenseCache(client).getFragmentRef(
      [id, options.fragment, cache.canonicalStringify(variables)],
      client,
      tslib.__assign(tslib.__assign({}, options), {
        variables: variables,
        from: id,
      }),
    );
