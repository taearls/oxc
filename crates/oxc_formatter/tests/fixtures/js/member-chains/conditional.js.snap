---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
id === null
  ? null
  : internal.getSuspenseCache(client).getFragmentRef(
      [id, options.fragment, cache.canonicalStringify(variables)],
      client,
      tslib.__assign(tslib.__assign({}, options), {
        variables: variables,
        from: id,
      }),
    );

==================== Output ====================
id === null
  ? null
  : internal.getSuspenseCache(client).getFragmentRef(
      [id, options.fragment, cache.canonicalStringify(variables)],
      client,
      tslib.__assign(tslib.__assign({}, options), {
        variables: variables,
        from: id,
      }),
    );

===================== End =====================
